formfactory/factory.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+
+
+class FormFactory(forms.Form):
+    """
```

Comment:
Stick to this format:
class Foo:
    """Bla bla docstring
    """

I think it work better with Sphynx. I may be wrong though.